v2/index.html

```diff
@@ -1358,14 +1358,40 @@ <h4>Chords</h4>
             <div class="issue" data-number="60"></div>
             <div class="issue" data-number="65"></div>
             <p>
-                Enter chords from the highest to the lowest note, each one separated by <code>^</code>.
+                Notes in a chords are enclosed in the characters <code>^</code> and <code>></code>.
+            </p>
+            <p>
+               A <a>duration</a> value MAY precede the <code>^</code>. Two or more <a>note names</a>
+               (with optional octave and accidental indications) MUST immediately follow this character.
+               The notes MUST be ordered from the lowest to the highest on the staff line. A <code>></code>
+               MUST be used to end the chord. A duration value for the individual notes within a chord MUST
+               NOT be supplied.
+            </p>
+            <p>
+                If the duration is omitted, the last specified duration is used. If no duration is
+                supplied in the <a>encoding</a> a default duration of <code>4</code> is assumed.
+            </p>
+            <p>
+                A <code>p</code> character MAY follow the duration to indicate the chord has a fermata.
```

Comment:
I guess we also want a trill? Not very likely, but I am sure somebody will find one

Reply:
I've created #148 -- I'd like to get this PR merged before it gets too far out of sync with `main` but yes, a trill should probably be allowed.